ocean_lib/test/test_exceptions.py

```diff
@@ -51,16 +51,10 @@ def test_AquariusError(publisher_ocean_instance, metadata):
     metadata_copy = metadata.copy()
```

Comment:
By the way, this function is already used for testing the Aquarius related exception. Would it be redundant if we add an unit test for `ddo_exists`?

Reply:
I think `ddo_exists` is sufficiently tested by the existing `test_AquariusError` unit test.